yw_website/website/views.py

```diff
@@ -30,51 +25,32 @@ def home(request):
     documents = paginator.get_page(page)
     return render(request, 'pages/home_page.html', { 'document_list': documents })
 
-def myworkflows(request):
-    documents_list = Workflow.objects.all()
+def my_workflows(request):
+    # TODO: Handle unathenticated user? Right now will just load an empty table.
+    documents_list = Workflow.objects.all().filter(user=request.user)
     for document in documents_list:
         latest_version = Version.objects.filter(workflow_id=document.id).order_by('last_modified').first()
         document.graph = latest_version.yw_graph_output if latest_version is not None else ""
     
     paginator = Paginator(documents_list, 10)
     page = request.GET.get('page')
     documents = paginator.get_page(page)
-    return render(request, 'pages/myworkflows_page.html', { 'document_list': documents })
-
-def model_form_upload(request):
-    if request.method == 'POST':
-        form = DocumentForm(request.POST, request.FILES)
-        form2 = ImageUploadForm(request.POST, request.FILES)
-        if form.is_valid():
-            form.save()
-            if form2.is_valid():
-                m = ExampleModel.objects.get(pk=course_id)
-                m.workflow = form.cleaned_data['image']
-                form2.save()
-            return redirect('home')
-    else:
-        form = DocumentForm()
-    return render(request, 'script_upload/upload_form.html', {
-        'form': form
-    })
+    return render(request, 'pages/home_page.html', { 'document_list': documents })
```

Comment:
my_workflows page works, change this back to render `pages/my_workflows_page.html`